packages/xod-arduino/src/transpiler.js

```diff
@@ -94,6 +94,11 @@ const arrangeTPatchesInTopologicalOrder = def(
     )(project)
 );
 
+const commentXodPragmas = def(
+  'commentXodPragmas :: String -> String',
+  R.replace(/#\s*pragma XOD/, '//#pragma XOD')
```

Comment:
The following should not be commented out:

```cpp
#pragma           XODULAR
```

Note multiple spaces and a custom continuation